packages/itinerary-body/src/TransitLegBody/alerts-body.tsx

```diff
@@ -131,6 +138,25 @@ export default function AlertsBody({
                       }}
                     />
                   )}
+                  {alertUrl.trim() && (
+                    <S.TransitAlertExternalLink href={alertUrl} target="_blank">
+                      <ExternalLinkAlt height={10} />
+                      <FormattedMessage
+                        defaultMessage={
+                          defaultMessages[
+                            "otpUi.TransitLegBody.AlertsBody.alertLinkText"
+                          ]
+                        }
+                        description="Text with the date an alert takes effect"
+                        id="otpUi.TransitLegBody.AlertsBody.alertLinkText"
+                        values={{ agency: agencyName || defaultAgencyName }}
+                      />
+                      <S.InvisibleAdditionalDetails>
+                        {" "}
+                        <FormattedMessage id="otpUi.TransitLegBody.AlertsBody.externalLink" />
```

Comment:
In this context, I feel that "external link" is redundant with "See alert on agency's website", unless it means "opens a new tab/window".

Reply:
Good point!